ord_schema/message_helpers.py

```diff
@@ -26,16 +26,15 @@
 import pandas as pd
 import requests
 from google import protobuf  # pytype: disable=import-error
-from google.protobuf import json_format
 from google.protobuf import text_format  # pytype: disable=import-error
+from google.protobuf import json_format
 from rdkit import Chem
 from rdkit.Chem import rdChemReactions
 from werkzeug import security
 
 import ord_schema
 from ord_schema import units
-from ord_schema.proto import dataset_pb2
-from ord_schema.proto import reaction_pb2
+from ord_schema.proto import dataset_pb2, reaction_pb2
```

Comment:
generally when importing modules they should stay as separate imports

Reply:
Resolved in [d77d77a](https://github.com/open-reaction-database/ord-schema/pull/701/commits/d77d77a3a92e8dbc3b8d17b6e8633ed6a9d4e7f8)